Wikia/selenium-tests
# Concf 626

https://wikia-inc.atlassian.net/browse/CONCF-626 Original bug is the parent ticket

changes:
2 selenium test that add youtube video to article using youtube tag
CreationTicket annotation that related a test to its ticket id
Modified video and videoprovider so that it supports sending youtube ID

Test cases covered:

Test Case 1:
1. Get a video id from youtube via API or hardcoded ID. The ID appears in url after v"="
2. put tag 
<youtube> 
ID 
</youtube> to article in source mode
3. Click preview 
4. Verify that the video appears on preview

Test Case 2:
1. Get a video id from youtube via API or hardcoded ID. The ID appears in url after v"="
2. put multiline tag 
<youtube>
HXnGWKdRIOs
</youtube>  to article in source mode
3. Click Publish
4. Verify that the video appears in the article
